hnn_core/tests/test_network.py

```diff
@@ -389,7 +411,7 @@ def test_network():
     # Test removing connections from net.connectivity
     # Needs to be updated if number of drives change in preceeding tests
     net.clear_connectivity()
-    assert len(net.connectivity) == 18
+    assert len(net.connectivity) == 50
```

Comment:
I am confused what's going on here. Isn't `clear_connectivity` going to remove all the connections? Then why do we still have 50 connections?

Reply:
`clear_connectivity` clears local network connectivity, not the drive connectivity. Maybe we should update the name accordingly.

Reply:
yeah updating the name sounds good. Either in this PR or another, as you feel like.